frontend/src/components/FlashMessage.js

```diff
@@ -0,0 +1,22 @@
+
+
+export const FlashMessage = ({ messageState, returnError }) => {
+    if (messageState) {
+        return (
+            <div className="container mb-4" style={{ backgroundColor: 'green', color: 'snow' }}>
```

Comment:
I think you should be using the alert alert-success and alert alert-danger classes for success/failure, but also I feel like the FlashMessage should only apply to successes. E.g. Successfully logged in, successfully added item, successfully deleted list...

Errors should still be local to the area where they occured, e.g. input required errors